libtelemetry/src/main/java/com/mapbox/android/telemetry/LocationEvent.java

```diff
@@ -35,8 +35,11 @@ public class LocationEvent extends Event implements Parcelable {
   private String applicationState;
   @SerializedName("horizontalAccuracy")
   private Float accuracy = null;
+  @SerializedName("permissionStatus")
+  private final String permissionStatus;
 
-  public LocationEvent(String sessionId, double latitude, double longitude, String applicationState) {
```

Comment:
nit: as this is a public API, adding a new parameter would be a breaking change.